app/panel/components/BuildingBlocks/StatsGraph.jsx

```diff
@@ -161,6 +161,15 @@ class StatsGraph extends React.Component {
 				.attrTween('stroke-dasharray', interpolator);
 		}
 
+		function getThemeColor(theme) {
+			switch (theme) {
+				default:
+					return '#124559';
+				case 'palm-theme':
+					return '#172a0b';
+			}
```

Comment:
Ignore my last comment, the early returns makes this work 😬I'm dumb, but regardless, can you swap the order of these statemetns, switches read a bit clearer when the default case is the last one

Reply:
Oh wow, I didn't know it worked like that. I'll always put default on the bottom then. Either way, good find